test/lsp.py

```diff
@@ -889,22 +900,31 @@ def wait_for_diagnostics(self, solc: JsonRpcProcess) -> List[dict]:
 
         return sorted(reports, key=lambda x: x['uri'])
 
+    def normalizeUri(self, uri, no_suffix = True):
```

Comment:
Second parameter is **never** used so it seems? Did I overlook it?

Reply:
Ah, there was a function I wanted to use it but then decided otherwise